plugin/publisher/snap-publisher-file/main_test.go

```diff
@@ -44,20 +43,18 @@ func TestFilePublisherLoad(t *testing.T) {
 	// These tests only work if SNAP_PATH is known.
 	// It is the responsibility of the testing framework to
 	// build the plugins first into the build dir.
-	if SnapPath != "" {
-		// Helper plugin trigger build if possible for this plugin
-		helper.BuildPlugin(PluginType, PluginName)
-		//
+
+	SkipConvey("make sure plugin has been built", t, func() {
+		err := helper.CheckPluginBuilt(SnapPath, PluginName)
+		So(err, ShouldBeNil)
 		//TODO cannot test this locally. We need AMQP and integration tests.
-		SkipConvey("ensure plugin loads and responds", t, func() {
+		SkipConvey("ensure plugin loads and responds", func() {
```

Comment:
I don't think these need to be SkipConvey's. [SkipConvey](https://github.com/smartystreets/goconvey/wiki/Skip) has the effect that the test is not run, we should be able to still verify that the plugin loads at the minimum.

I know that's what was there before but I think it makes sense to test that plugin is loadable as a minimal test.